iothub/device/src/Transport/AmqpIot/AmqpIotMessageConverter.cs

```diff
@@ -30,7 +30,7 @@ internal class AmqpIotMessageConverter
 
         private const string MethodName = "IoThub-methodname";
         private const string Status = "IoThub-status";
-        private const string FailedToSerializeUnsupportedType = "FailedToSerializeUnsupportedType";
+        private const string FailedToSerializeUnsupportedType = "FailedToSerializeUnsupportedType: {0}";
```

Comment:
Can this be written like normal English? "Failed to serialize an unsupported type of '{0}'."